tests/test_encoders.py

```diff
@@ -1060,9 +1060,10 @@ def test_round_trip(self):
         """Test an encoding round-trip"""
         ds = get_testdata_file("MR_small_RLE.dcm", read=True)
         arr = ds.pixel_array
-        del ds.PixelData
+        ds.PixelData = None
+        ds._pixel_array = None
```

Comment:
Hopefully fixed a weird intermittent failure where somehow the `arr` and `ds.pixel_array` ended up wit the same ID. Still not completely sure why, maybe it was reusing the same block of memory?